modules/nw-ovn-kubernetes-alerts-cli.adoc

```diff
@@ -26,14 +26,11 @@ $ ALERT_MANAGER=$(oc get route alertmanager-main -n openshift-monitoring \
 -o jsonpath='{@.spec.host}')
 ----
 
-.. Issue a `curl` request to the alert manager route API with the correct authorization details requesting specific fields by running the following command:
+.. Issue a `curl` request to the alert manager route API by running the following command, replacing the `$ALERT_MANAGER` with the URL of your `Alertmanager` instance:
 +
 [source,terminal]
 ----
-$ curl -s -k -H "Authorization: Bearer \
-$(oc create token prometheus-k8s -n openshift-monitoring)" \
-https://$ALERT_MANAGER/api/v1/alerts \
-| jq '.data[] | "\(.labels.severity) \(.labels.alertname) \(.labels.pod) \(.labels.container) \(.labels.endpoint) \(.labels.instance)"'
+$ curl -s -k -H "Authorization: Bearer $(oc create token prometheus-k8s -n openshift-monitoring)" https://$ALERT_MANAGER/api/v1/alerts | jq '.data[] | "\(.labels.severity) \(.labels.alertname) \(.labels.pod) \(.labels.container) \(.labels.endpoint) \(.labels.instance)"'
```

Comment:
From the docs portal, this is copied as what I've changed, but our docs currently show it as the diff (red) that I changed. @tssurya could you check the syntax of this command, please ? I just noticed a difference and am not an engr.

Reply:
IC didnt change how to curl alerts endpoint from kapi. This is just a format change and the commands look identical to me.